data_validation/cli_tools.py

```diff
@@ -954,6 +955,16 @@ def _add_common_arguments(optional_arguments, required_arguments):
     )
 
 
+def _check_no_partitions(value):
```

Comment:
Is this value a string? If so, lets annotate it
```suggestion
def _check_no_partitions(value: str) -> int:
```